api/app/serializers/text_serializer.rb

```diff
@@ -6,7 +6,7 @@ class TextSerializer < ApplicationSerializer
              :published, :annotations_count, :highlights_count, :bookmarks_count,
              :age, :position, :publication_date, :cover_styles, :subtitle,
              :slug, :section_kind, :title_formatted, :title_plaintext,
-             :subtitle_formatted, :subtitle_plaintext, :description
+             :subtitle_formatted, :subtitle_plaintext, :description, :toc
```

Comment:
The `toc` attribute from https://github.com/ManifoldScholar/manifold/blob/master/api/app/serializers/text_full_serializer.rb#L7 can also be removed as part of this.  

Sometimes for big models we have two serializers: a partial one and a full one.  In cases where we just need some data for a model, we prefer the partial to speed things up.  The full serializer inherits everything defined in the partial one, so if an attribute is moved from one, it doesn't need to be defined in the other as well.

Reply:
OK nice! thanks for the explanation.